### ISSUE TITLE
Client crashes on full screen after input capture on Wayland

### ISSUE THREAD

**Issue by quietvoid** [gnif/LookingGlass#540]
### Bug Report Required Information

The entire (not truncated) output from the client application (if applicable).
To obtain this run `looking-glass-client` in a terminal.

```
 48722745628 [I]               main.c:1029 | main                           | Looking Glass (B3-178-gfd531bd39b)
 48722745657 [I]               main.c:1030 | main                           | Locking Method: Atomic
 48722746975 [I]            ivshmem.c:126  | ivshmemOpenDev                 | KVMFR Device     : /dev/shm/looking-glass
 48722767621 [I]                egl.c:273  | egl_initialize                 | Double buffering is off
 48722767626 [I]               main.c:640  | tryRenderer                    | Using Renderer: EGL
 48722767800 [I]                 gl.c:57   | waylandGetEGLDisplay           | Using eglGetPlatformDisplay
 48722792340 [I]                egl.c:684  | egl_render_startup             | Single buffer mode
 48722796113 [I]                egl.c:700  | egl_render_startup             | EGL       : 1.5
 48722796118 [I]                egl.c:701  | egl_render_startup             | Vendor    : AMD
 48722796120 [I]                egl.c:702  | egl_render_startup             | Renderer  : Radeon RX 5500 XT (NAVI14, DRM 3.40.0, 5.12.5-arch1-1, LLVM 11.1.0)
 48722796123 [I]                egl.c:703  | egl_render_startup             | Version   : OpenGL ES 3.2 Mesa 21.1.0
 48722796126 [I]                egl.c:704  | egl_render_startup             | EGL APIs  : OpenGL OpenGL_ES 
 48722796127 [I]                egl.c:705  | egl_render_startup             | Extensions: EGL_ANDROID_blob_cache EGL_ANDROID_native_fence_sync EGL_EXT_buffer_age EGL_EXT_create_context_robustness EGL_EXT_image_dma_buf_import EGL_EXT_image_dma_buf_import_modifiers EGL_EXT_swap_buffers_with_damage EGL_KHR_cl_event2 EGL_KHR_config_attribs EGL_KHR_create_context EGL_KHR_create_context_no_error EGL_KHR_fence_sync EGL_KHR_get_all_proc_addresses EGL_KHR_gl_colorspace EGL_KHR_gl_renderbuffer_image EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_3D_image EGL_KHR_gl_texture_cubemap_image EGL_KHR_image_base EGL_KHR_no_config_context EGL_KHR_reusable_sync EGL_KHR_surfaceless_context EGL_KHR_swap_buffers_with_damage EGL_EXT_pixel_format_float EGL_KHR_wait_sync EGL_MESA_configless_context EGL_MESA_drm_image EGL_MESA_image_dma_buf_export EGL_MESA_query_driver EGL_WL_bind_wayland_display EGL_WL_create_wayland_buffer_from_image 
 48722804446 [I]                 gl.c:83   | waylandEGLSwapBuffers          | Using EGL_KHR_swap_buffers_with_damage
 48722810853 [ ]               main.c:865  | lg_run                         | ================================================================================
 48722810859 [I]               main.c:866  | lg_run                         | The host application seems to not be running
 48722810860 [I]               main.c:867  | lg_run                         | Waiting for the host application to start...
 48722870979 [I]               main.c:922  | lg_run                         | Host ready, reported version: B3-178-gfd531bd39b
 48722870989 [I]               main.c:923  | lg_run                         | Starting session
 48723047448 [I]               main.c:470  | main_frameThread               | Format: FRAME_TYPE_BGRA 2560x1440 stride:2560 pitch:10240 rotation:0
zwp_pointer_constraints_v1@11: error 1: a pointer constraint with a wl_pointer of the same wl_seat is already on this surface
```

Reproduction steps:
- Open looking-glass
- Press scroll lock to capture
- Press scroll lock + F

Version: git master @ fd531bd39b
Using Sway.

**Comment by quantum5** [comment]
Can you try out #541 and see if it fixes the issue?

**Comment by quietvoid** [comment]
@quantum5 Yep, it fixes it.

**Comment by quietvoid** [comment]
This issue is happening in master on facbb54776dde77d56a0494b270f57d32f8d9e5b again, but I'm not able to reopen the issue.

**Comment by quantum5** [comment]
Can you see if #570 fixes?

**Comment by quietvoid** [comment]
Yes, the issue is fixed by d43ddfc.
Thank you.
